docs/integrations/salesforce/ssowithendpoints.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Secure ngrok Endpoints using Salesforce OpenID Connect
+description: Use Salesforce's managed applications to add SSO for your ngrok resources
+---
+
+In this guide, you will use Salesforce as an OAuth provider along with the ngrok OIDC traffic policy action to secure your ngrok resources. You will use Salesforce as the primary IdP for your endpoints and the OIDC traffic policy action will supplement an ID token that will be used to identify users that log in to your endpoint.
+
+Since Salesforce is currently not a supported OAuth provider by ngrok, this will allow you to leverage its OAuth capabilities and add an extra layer of authorization to your endpoints.
+
+In summary:
+
+#### Through Salesforce you will obtain:
+
+- An external client application issuer URL
+- A client ID that will be used in your traffic policy
+- A client secret that will be used in your traffic policy
+
+#### Through ngrok’s OIDC flow you will obtain:
+
+- An ID token that can be used to identify users
+- The ability to route and restrict access at a more granular level based on user metadata
```

Comment:
```suggestion
```

I actually don't think we need any of this, as the user will understand these things by following along the guide